scripts/todo.js

```diff
@@ -335,52 +323,61 @@ window.onload = function() {
     }
 
     return true;
-  }
+  };
 
-  // wire up notification permission functionality to "Enable notifications" button
+  // wire up notification permission functionality to 'Enable notifications' button
 
   notificationBtn.addEventListener('click', askNotificationPermission);
 
   function createListItem(contents) {
     const listItem = document.createElement('li');
     listItem.textContent = contents;
     return listItem;
-  }
+  };
 
   // function for creating the notification
   function createNotification(title) {
 
     // Create and show the notification
-    let img = '/to-do-notifications/img/icon-128.png';
-    let text = 'HEY! Your task "' + title + '" is now overdue.';
-    let notification = new Notification('To do list', { body: text, icon: img });
+    const img = '/to-do-notifications/img/icon-128.png';
+    const text = `HEY! Your task "${title}" is now overdue.`;
+    const notification = new Notification('To do list', { body: text, icon: img });
 
-    // we need to update the value of notified to "yes" in this particular data object, so the
+    // we need to update the value of notified to 'yes' in this particular data object, so the
     // notification won't be set off on it again
 
     // first open up a transaction as usual
-    let objectStore = db.transaction(['toDoList'], "readwrite").objectStore('toDoList');
+    const objectStore = db.transaction(['toDoList'], 'readwrite').objectStore('toDoList');
 
     // get the to-do list object that has this title as it's title
-    let objectStoreTitleRequest = objectStore.get(title);
+    const objectStoreTitleRequest = objectStore.get(title);
 
     objectStoreTitleRequest.onsuccess = function() {
       // grab the data object returned as the result
-      let data = objectStoreTitleRequest.result;
+      const data = objectStoreTitleRequest.result;
 
-      // update the notified value in the object to "yes"
-      data.notified = "yes";
+      // update the notified value in the object to 'yes'
+      data.notified = 'yes';
 
       // create another request that inserts the item back into the database
-      let updateTitleRequest = objectStore.put(data);
+      const updateTitleRequest = objectStore.put(data);
 
       // when this new request succeeds, run the displayData() function again to update the display
       updateTitleRequest.onsuccess = function() {
         displayData();
-      }
-    }
-  }
+      };
+    };
+  };
 
   // using a setInterval to run the checkDeadlines() function every second
   setInterval(checkDeadlines, 1000);
 }
+
+const ordinal = (day) => {
```

Comment:
This is not an anonymous function. Why not this?
```suggestion
function ordinal(day) {
```

Reply:
because it is not using `this` context